NDB.Covid19/NDB.Covid19.Droid/Views/AuthenticationFlow/LoadingPageActivity.cs

```diff
@@ -69,15 +76,23 @@ void OnActivityFinished()
             
         }
 
-        void OnError(Exception e)
+        void OnError(Exception e, bool isOnFail = false)
         {
             if (e is AccessDeniedException)
             {
-                LogUtils.LogMessage(LogSeverity.INFO, "The user refused to share keys", null);
+                LogUtils.LogMessage(LogSeverity.INFO, "The user refused to share keys", null, GetCorrelationId());
                 RunOnUiThread(() => { GoToInfectionStatusPage(); });
             }
             else
             {
+                if (!isOnFail)
+                {
+                    LogUtils.LogMessage(
+                        LogSeverity.INFO,
+                        "Something went wrong during key sharing (INFO with correlation id)",
```

Comment:
remove the "(INFO with correlation id)" it is a bad legacy code

Reply:
Has been removed.